### ISSUE TITLE
Default for `--work-dir`

### ISSUE THREAD

**Issue by marcodelapierre** [seqeralabs/tower-agent#47]
The current default is `~/work`.

I am wondering whether the user experience might benefit from using instead `$TW_AGENT_WORK`.
It seems to me like this variable is used as default work directory within the Tower web interface; hence, adopting it for the Agent would align the two, increase consistency, and in turn user friendliness (hopefully).

Disclaimer: I have been using Nextflow for years, but have just started poking around with Tower!

**Comment by jordeu** [comment]
Currently at Tower the variable `$TW_AGENT_WORK` do not refer to any environment variable, it refers to the value that the user has set the `--work-dir` parameter when launching the Tower Agent. Given that several users can use the same Compute Environment with his own agent, this value can be different for each user.

When Tower Agent is run without `--work-dir` parameter then it defaults to `~/work`. Do you propose to default to `TW_AGENT_WORK` environment variable if defined?

**Comment by marcodelapierre** [comment]
yes exactly, I am suggesting TW_AGENT_WORK as default, if defined within the shell session, and otherwise fall back to ~/work. --work-dir would still be available to override the default

**Comment by jordeu** [comment]
I like this idea. I'll implement ASAP.
